harrisonDir/harrisonHomework.txt

```diff
@@ -0,0 +1,271 @@
+1.) 
+
+var properFrac = function(n,d) {
+  if (n >= d && n%d != 0) {				/* If it's improper and n is not divisible by d*/
+    var wholeNum = ( Math.floor(n/d) );		/* Number of times n goes into d completely */
+    var remainder = n%d;
+    return ( wholeNum + " " + remainder + "/" + d ); 
+  } else if ( Math.floor(n/d) === 0) {
+      return (n/n) + "/" + (d/n);			/* If n goes into d evenly, simplify */
+  } else {								/* Any other cases should be proper fractions, so just put a slash between them */
+    	return (n + "/" + d);
+  }
+};
+
```

Comment:
Your code on this one is good but I think the logic is going in a different route than the problem needs. You are doing a little more work than is necessary. 

Check out Dan's answer:
function fractionString(n,d) {
  var remainder = n%d,
      wholes = Math.floor(n/d);
            // OR: (n - remainder)/d; //always integer
  return wholes + ' ' + remainder + '/' + d;
}